### ISSUE TITLE
Link button not working in IE

### ISSUE THREAD

**Issue by adrigen** [plone/Products.TinyMCE#86]
It has been reported to me from two people in two different companies, that the link button does nothing whilst using IE. 

I do not know the version of the first instance, but the most recent was ie11. 
I have tried to replicate this without any success in a VM. 

I though that since it was so common in my experience, someone else may have seen this issue too?

**Comment by allcaps** [comment]
Yes. Bumped into the same on IE11. Forcing IE11 into IE10 mode worked for me. In main_template.pt:

    dummy python:request.RESPONSE.setHeader('X-UA-Compatible', 'IE=10,chrome=1');

**Comment by djay** [comment]
see https://dev.plone.org/ticket/20131 and https://dev.plone.org/ticket/13834

**Comment by mauritsvanrees** [comment]
Those dev.plone.org links eventually lead to pull request #105 which was merged and has been included since 1.3.10, so maybe this was fixed already.
